acmw-db-app/pages/api/attendance/[pid].js

```diff
@@ -26,7 +26,16 @@ async function record (event_code, f_name, l_name_dot_num, year_level) {
        curr_student_id = await createStudent(f_name, l_name, l_name_dot_num, "", year_level, "");
    }
 
-    console.log(curr_student_id);
+    //school_level
+    // need to update student year level if it has changed
+    let current_student_yr = await pgQuery(`SELECT school_level FROM student WHERE name_dot_num = '${l_name_dot_num.toLowerCase()}'`);
```

Comment:
I wouldn't bother checking if it is different - I'd rather have one query in every case than possibly 2 queries in some cases. If it hasn't changed no damage is done with update.